ietf-layer0-types-ext.yang

```diff
@@ -51,4 +51,494 @@ module ietf-layer0-types-ext {
+                        for this lane";
+    }
+    leaf max-polarization-dependent-loss {
+           type decimal64 {
+             fraction-digits 2;
+           }
+           units dB ;
+           description "Polarization dependent loss";
+    }
+    leaf-list available-modulation-types {
+      type identityref {
+      base modulation;
+      }
+      config false;
+    description
+      "List of modulation types the tranceiver can support";
+    }
+    leaf  osnr-margin {
+      type int32;
+      units "dB";
+      config false;
+      description "OSNR margin to FEC threshold";
+    }
+    leaf  q-margin {
+      type int32;
+      units "dB";
+      config false;
+      description "Q-Factor margin to FEC threshold";
+    }
+    leaf-list available-baud-rates {
+      type uint32;
+        units Bd;
+      config false;
+      description
+        "list of available baud-rates. 
+         Baud-rate is the unit for
+         symbol rate or modulation rate 
+         in symbols per second or
+         pulses per second. 
+         It is the number of distinct symbol
+         changes (signal events) made to the 
+         transmission medium
+         per second in a digitally 
+         modulated signal or a line code";
+    }
+    leaf-list available-FEC-types {
+      type identityref {
+      base FEC;
+      }
+      config false;
+      description "List determining all the available FEC";
+    }
+    leaf FEC-code-rate {
+      type decimal64 {
+      fraction-digits 8;
+      range "0..max";
+      }
+      config false;
+      description "FEC-code-rate";
+    }
+    leaf FEC-threshold {
+      type decimal64 {
+      fraction-digits 8;
+      range "0..max";
+      }
+      config false;
+    description
+        "Threshold on the BER, for which FEC 
+         is able to correct errors";
+    }
+
+  } // end grouping of attributes capabilities related to operational mode
+
+/* This grouping represent the list of configured parameters */
+/* values independet of operational mode */
+
+  grouping common-tranceiver-configured-param {
+    description "Capability of an optical tranceiver";
+
+
+    leaf transmit-frequency {
+      type layer0-types:frequency-thz;
+      config false;
+      description
+        "his parameter indicates the actual configured transmitter
+         frequency ";
+    }
+
+    leaf channel-output-power {
+        type dbm-t;
+        description "Current transmit power";
+    }
+
+    leaf channel-input-power {
+        type dbm-t;
+        config false;
+        description "The current OTSi input power ";
+    }
+
+    leaf total-input-power {
+        type dbm-t;
+        config false;
+        description "Current total receive power";
+    }
+
+  } // grouping for configured attributes out of mode
+
+
+/* This grouping represent the list of parameters values linked to operational-mode  */ 
+
+  grouping common-tranceiver-configured-param-bymode {

```

Comment:
I do not think we need this grouping: we need only the configured mode in the common-tranceiver-configured-param grouping

Reply:
**2020-08-13 CCAMP**

@sergiobelotti : remove this grouping

Reply:
solve with commit https://github.com/ietf-ccamp-wg/ietf-ccamp-layer0-types-ext/pull/4/commits/40cd7b90dc7e0a65e6bb0b60800e9ae83e455b6c